README.md

```diff
@@ -76,6 +76,30 @@ the elements of the variable will be initialized to `0, 1, 2, 3, 4, ...`,
 depending on the number of elements in `T`. For arithmetic types 
 `vir::iota_v<T>` is always just `0`.
 
+### Making `simd` conversions more convenient:
+
+The TS is way too strict about conversions, requiring verbose 
+`std::experimental::static_simd_cast<T>(x)` instead of a concise `T(x)` or 
+`static_cast<T>(x)`. (`std::simd` in C++26 will fix this.)
+
+```c++
+#include <vir/simd_cvt.h>
+
+using floatv = stdx::native_simd<float>;
+using intv = stdx::rebind_simd_t<int, floatv>;
+
+void f(intv x) {
+  using vir::cvt;
+  x = cvt(10 * sin(floatv(cvt(x))));
+  // without vir::cvt:
+  x = stdx::static_simd_cast<intv>(10 * sin(stdx::static_simd_cast<floatv>(x)));
+}
```

Comment:
The example should show how to assign to a new local variable?

What would work?
```cpp
// Probably not? What type would y1 have?
auto y1 = cvt(10 * sin(floatv(cvt(x))));
// Does that work, hopefully?
intv y2 = cvt(10 * sin(floatv(cvt(x))));
// I'd expect this to work?
auto y3 = intv(cvt(10 * sin(floatv(cvt(x)))));
// This is probably an error?
auto y3 = cvt<intv>(10 * sin(floatv(cvt(x))));
```

Reply:
`y1`-`y3` work. I'll document `y1`. `y3` I already answered below.